<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-21T07:45:29 IST" name="Suite" finished-at="2023-11-21T07:45:42 IST" duration-ms="12769">
    <groups>
    </groups>
    <test started-at="2023-11-21T07:45:29 IST" name="Test" finished-at="2023-11-21T07:45:42 IST" duration-ms="12769">
      <class name="testcases.RegisterTestCases">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:testcases.RegisterTestCases@6cd28fa7]" started-at="2023-11-21T07:45:29 IST" name="launchBrowser" finished-at="2023-11-21T07:45:33 IST" duration-ms="3674" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="verifyRegisterationWithInBlankData()[pri:0, instance:testcases.RegisterTestCases@6cd28fa7]" started-at="2023-11-21T07:45:33 IST" name="verifyRegisterationWithInBlankData" finished-at="2023-11-21T07:45:36 IST" duration-ms="2595" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegisterationWithInBlankData -->
        <test-method signature="verifyRegisterationWithValidData()[pri:0, instance:testcases.RegisterTestCases@6cd28fa7]" started-at="2023-11-21T07:45:36 IST" name="verifyRegisterationWithValidData" finished-at="2023-11-21T07:45:39 IST" depends-on-methods="testcases.RegisterTestCases.verifyRegisterationWithInBlankData" duration-ms="2987" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://naveenautomationlabs.com/opencart/index.php?route=account/success] but found [https://naveenautomationlabs.com/opencart/index.php?route=account/register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://naveenautomationlabs.com/opencart/index.php?route=account/success] but found [https://naveenautomationlabs.com/opencart/index.php?route=account/register]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.Resources.commonMethods.handleAssertions(commonMethods.java:18)
at testcases.RegisterTestCases.verifyRegisterationWithValidData(RegisterTestCases.java:80)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegisterationWithValidData -->
      </class> <!-- testcases.RegisterTestCases -->
      <class name="testcases.LoignTestCases">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:testcases.LoignTestCases@543588e6]" started-at="2023-11-21T07:45:39 IST" name="launchBrowser" finished-at="2023-11-21T07:45:41 IST" duration-ms="2759" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="validLoginTest()[pri:0, instance:testcases.LoignTestCases@543588e6]" started-at="2023-11-21T07:45:41 IST" name="validLoginTest" finished-at="2023-11-21T07:45:42 IST" duration-ms="712" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLoginTest -->
      </class> <!-- testcases.LoignTestCases -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
